src/main/resources/docker/Dockerfile

```diff
@@ -7,7 +7,8 @@ RUN groupadd -g 1000 -r sld \
  && useradd -u 1000 -g 1000 -r -m sld \
  && chmod +x /data/*.sh \
  && mkdir /var/lm \
- && chown sld:sld /var/lm /data
+ # set group to 0 to allow arbitrary users in OCP to write to this directory
+ && chown -R sld:0 /var/lm /data
```

Comment:
Can the HOME env variable be set similarly to the ansible lifecycle driver? https://github.com/IBM/ansible-lifecycle-driver/blob/0593142d4a94ada974a1d3750dde4a23fa4f7371/docker/Dockerfile#L35?

That allows the arbitrary user to have a home directory.
Also set permissions and ownership of the home directory.

Reply:
Yes, makes sense. I'll make the change.

Reply:
Ok, change committed.